internal/apischema/openai/openai.go

```diff
@@ -977,7 +977,12 @@ type ChatCompletionResponseChoiceMessage struct {
 	ToolCalls []ChatCompletionMessageToolCallParam `json:"tool_calls,omitempty"`
 
 	// Annotations for the message, when applicable, as when using the web search tool.
-	Annotations []Annotation `json:"annotations,omitempty"`
+	//
+	// Note: do not use the value of slice, but instead use a pointer to the slice since otherwise,
```

Comment:
Maybe move this to a unit test that proves the round trip concern? Then you can shorten the comment and be sure it wont be undone unless someone deletes the test